### DISCUSSION TITLE
New Zigbee2mqtt device support

### DISCUSSION THREAD

**Discussion by kamaldeepdhiman** [Koenkk/zigbee2mqtt#21309] [Devices]
hi there, can any one help me to all new zigbee device to zigbee2mqtt network. I tried a lot but unsuccessful. my device generated_external_definition are as below: not supported by zigbee2mqtt network at this stage but able to join the network. please help me.

const {deviceEndpoints, onOff} = require('zigbee-herdsman-converters/lib/modernExtend');

const definition = {
    zigbeeModel: ['LXN56-TS27LX1.2'],
    model: 'LXN56-TS27LX1.2',
    vendor: '3A Smart Home DE',
    description: 'Automatically generated definition',
    extend: [deviceEndpoints({"endpoints":{"1":1,"2":2}}), onOff({"powerOnBehavior":false,"endpoints":{"1":1,"2":2}})],
    meta: {"multiEndpoint":true},
};

module.exports = definition;

**Comment by pcurt** [comment]
Hi,
I'm trying to do the same thing,
I don't understand why this part is generated : `deviceEndpoints({"endpoints":{"1":1,"2":2}})` 
This function doesn't exist in `zigbee-herdsman-converters/lib/modernExtend` file.
